docker/start_postgres.sh

```diff
@@ -13,12 +15,26 @@ if [[ "$ACTION" == "start" ]]; then
     fi
 fi
 
+su postgres -c 'pg_ctl status' >/dev/null 2>&1
+if [[ $? == 4 ]]; then
+    echo "Making new postgres database at ${PGDATA}"
+    mkdir -p $PGDATA
+    chown -f postgres:postgres $PGDATA
+    su postgres -c "initdb" >/dev/null 2>&1
+    INIT_RESOLVE=0
+    ACTION=start
+fi
+
 # Only locally start postgres if we weren't given a PG_HOST environment variable
 if [[ -z "$PG_HOST" ]]; then
-  su postgres -c "pg_ctl --silent --pgdata=/var/postgres --log=/var/postgres/postgresql.log $ACTION"
+  su postgres -c "pg_ctl --silent --log=${PGDATA}/postgresql.log ${ACTION}"
 fi
 
 if [[ "$ACTION" == "start" ]]; then
     # wait for postgres to start (local or PG_HOST)
     until pg_isready -q ; do sleep 1 ; done
 fi
+
+if [[ $INIT_RESOLVE ]]; then
+    su postgres -c "createuser -s root"
+fi
```

Comment:
Nit, trailing newline:
```suggestion
fi

```